src/plugins/gr-he-common/core/shell.py

```diff
@@ -103,5 +109,34 @@ def _setup(self):
                 if not self.environment[ohostedcons.CoreEnv.TMUX_PROCEED]:
                     raise context.Abort('Aborted by user')
 
+        if (
+            not self.environment[ohostedcons.NetworkEnv.FORCE_IPV4]
+            and not self.environment[ohostedcons.NetworkEnv.FORCE_IPV6]
+        ):
+            dialog.queryEnvKey(
+                dialog=self.dialog,
+                logger=self.logger,
+                env=self.environment,
+                key=ohostedcons.CoreEnv.FORCE_IP_PROCEED,
+                name='FORCE_IP_PROCEED',
+                note=_(
+                    '\nIt has been detected that either --4 or --6 '
```

Comment:
I think:

It has been detected that neither --4 nor --6 options were used

Or:

It has been detected that neither the option --4 nor --6 was used